### PR TITLE
Release 1.5 backports

### PR DESCRIPTION
**Pull request by sdodson**
picks #3781 #3699 #3801

### CODE REVIEW

**Comment by sdodson** [comment]
aos-ci-test

File: openshift-ansible.spec [hunk 1]
```diff
@@ -250,7 +250,7 @@ BuildArch:     noarch
 %package -n atomic-openshift-utils
 Summary:       Atomic OpenShift Utilities
 BuildRequires: python-setuptools
-Requires:      %{name}-playbooks >= %{version}
+Requires:      %{name}-playbooks = %{version}
```
Comment (ingvagabund) [inline openshift-ansible.spec]: +1

**Comment by sdodson** [comment]
aos-ci-test

**Comment by sdodson** [comment]
aos-ci-test

**Comment by sdodson** [comment]
@jhadvig fyi

**Pull request merged**